### PR TITLE
6 as client i want total balance of my crypto

### PR DESCRIPTION
**Pull request by rubeniriso**
Issue #6 
See if I missed anything, please.
Thanks

### CODE REVIEW

File: tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+            ->expects("getWallet")
+            ->with($walletId)
+            ->once()
+            ->andThrow(new Exception('Wallet not found'));
+
+        $this->expectException(Exception::class);
+
+        $this->getWalletBalanceService->execute($walletId);
+    }
+
```
Comment (martinez2001) [inline tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php:45]: Not sure if \n needed. Check it.

File: tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            ->once()
+            ->andReturn($wallet);
+
+        $expectedWalletBalance = 0;
+
+        $response = $this->getWalletBalanceService->execute($walletId);
+
+        $this->assertEquals($expectedWalletBalance, $response);
+    }
+
```
Comment (martinez2001) [inline tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php:68]: same.

File: tests/app/Infrastructure/Controller/GetWalletBalanceControllerTest.php [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+            ->with('999')
+            ->once()
+            ->andThrow(new Exception('Wallet not found'));
+
+        $response = $this->get('api/wallet/999/balance');
+
+        $response->assertStatus(Response::HTTP_NOT_FOUND)
+            ->assertExactJson(['error' => 'a wallet with the specified ID was not found']);
+    }
+
```
Comment (martinez2001) [inline tests/app/Infrastructure/Controller/GetWalletBalanceControllerTest.php:43]: same.

**Review by martinez2001 — CHANGES_REQUESTED** [review]
Few format changes needed.

File: tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php (hunk 1 above)
Comment (rubeniriso) [reply tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php:45]: done.

File: tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php (hunk 2 above)
Comment (rubeniriso) [reply tests/app/Application/WalletBalance/GetWalletBalanceServiceTest.php:68]: same.

File: tests/app/Infrastructure/Controller/GetWalletBalanceControllerTest.php (hunk 3 above)
Comment (rubeniriso) [reply tests/app/Infrastructure/Controller/GetWalletBalanceControllerTest.php:43]: same.

**Review by martinez2001 — APPROVED** [review]
Ok to merge.

**Pull request merged**